main.py

```diff
@@ -26,28 +26,38 @@ def main_loop(
     controller -- the scene manager acting as the main controller of the game
     screen -- the screen on which the current frame rate is displayed
     clock -- the clock regulating the maximum frame rate of the game
+
+    Returns:
+    True -- restart game
+    False -- exit game
     """
-    quit_game: bool = False
+    quit_game: int = False
```

Comment:
This section needs to be improved.

We need to decide if `quit_game` is a `bool` or a `int`, it should not be both at different times.
Also, we should avoid giving unclear signification to arbitrary numbers.

Ideally, it should be using an `Enum`: [https://docs.python.org/3/library/enum.html]( https://docs.python.org/3/library/enum.html).
So then each different value that could be taken by the variable would having a nice label, and also it should not be possible to have value outside of the range of available ones.

Alternatively, constants need to be defined in `src/constants.py`.

Reply:
Understood, modified

Reply:
Looks way better!